api.swagger.yaml

```diff
@@ -2850,6 +2869,13 @@ components:
         - required:
             - email
             - username
+    user-metrics-accessed-schema:
+      type: object
+      properties:
+        'users who accessed the site on 4 or more days within a month':
```

Comment:
I know it's taken directly from the task but this is a really awful property name.